c-ares-sys/build/expando.c

```diff
@@ -0,0 +1,7 @@
+#include <ares_version.h>
+
+#define VERSION2(a, b, c) RUST_VERSION_C_ARES_##a##_##b##_##c
+#define VERSION(a, b, c) VERSION2(a, b, c)
```

Comment:
I guess my C must be rusty - if you'll forgive the pun.  What's the point of having both `VERSION` and `VERSION2`?

Reply:
to take the value of ARES_VERSION_MAJOR these macros

Reply:
I don't follow.  

Why isn't a single macro `define VERSION(a, b, c) RUST_VERSION_C_ARES_##a##_##b##_##c` enough?

Reply:
This would expand to
RUST_VERSION_C_ARES_ARES_VERSION_MAJOR_ARES_VERSION_MINOR_ARES_VERSION_PATCH,
where the value of ARES_ARES_VERSION_MAJOR is not expanded.